### PR TITLE
Craig student stats branch

### PR DESCRIPTION
**Pull request by Craigieboy10**
Student can now log in appropriately and view their taken quiz stats specific to them.

### CODE REVIEW

File: src/java/quizapp/bean/StudentQuizStat.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/*
```
Comment (Craigieboy10) [inline src/java/quizapp/bean/StudentQuizStat.java]: new bean to handle instances of a quiz stats for particular student

File: src/java/quizapp/controller/StudentResults.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            //gatherStaffProfile(staffLogin.getID(), request, response);
+        } else if (session.getAttribute("whoLog").equals("student")) {
+            StudentLogin studentLogin = (StudentLogin) session.getAttribute("StudentLogin");
+            gatherStudentResults(studentLogin.getID(), request, response);
+        }
+    }
+
+    private void gatherStudentResults(int matricN, HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        FetchStudentResults fsr = new FetchStudentResults();
+        request.setAttribute("statList", fsr.getStudentStats(matricN));
```
Comment (Craigieboy10) [inline src/java/quizapp/controller/StudentResults.java]: important code here places the list of beans for use in html forEach loop to output quiz results into a table

File: src/java/quizapp/controller/StudentResults.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        RequestDispatcher rd = request.getRequestDispatcher("/studentResults.jsp");
+        rd.forward(request, response);
```
Comment (Craigieboy10) [inline src/java/quizapp/controller/StudentResults.java]: everything works but url left with ? on tail end... thoughts?

File: test/quizapp/model/AddAccountTest.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
+/*
```
Comment (Craigieboy10) [inline test/quizapp/model/AddAccountTest.java]: jUnits I was researching over the weekend

File: web/studentResults.jsp [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+                                <th>${listItem.qs}</th>
+                                <th>${listItem.dt}</th>
+                            </tr>
+                        </c:forEach>
+                    </table>
+                </div>
+            </div>
+        </div>
+    </div>
+    <%@include file="footer.jsp"%>
```
Comment (Craigieboy10) [inline web/studentResults.jsp]: some nice industry standard code right here

**Comment by aGrimes94** [comment]
everything looks good!

**Pull request merged**